src/main/webapp/app/pages/curation/review/ReviewPage.tsx

```diff
@@ -50,6 +50,7 @@ const ReviewPage: React.FunctionComponent<IReviewPageProps> = (props: IReviewPag
   const [isAcceptingAll, setIsAcceptingAll] = useState(false);
 
   const [isAccepting, setIsAccepting] = useState(false);
+  const [isRejecting, setIsRejecting] = useState(false);
```

Comment:
Could we combine these two? It seems we just want to disable actions when this is happening?

Could we do something similar with isAcceptingAll? I see you hide the page when this is happening, but is it enough to just disable the buttons. If this is the case, we could remove these 3 state variables and combine them into a `loading` state

Reply:
I can combine the isAccepting and isRejecting into one status. However, we do need isAcceptingAll because one of the FirebaseGeneReviewService method needs to know that status.